doc/Examples.rst

```diff
@@ -231,6 +231,24 @@ sentence usually includes any trailing whitespace.
     // [14, 26)
 
 
+Layout Regions
+---------
```

Comment:
The style in this document is to have as many hyphens as characters in the heading.